packages/@react-spectrum/labeledvalue/src/LabeledValue.tsx

```diff
@@ -78,6 +78,15 @@ export const LabeledValue = React.forwardRef(function LabeledValue<T extends Spe
   } = props;
   let domRef = useDOMRef(ref);
 
+  // todo(sanmalik) - fix this
+  if (
+    domRef?.current &&
+    domRef.current.querySelectorAll("input, [contenteditable], textarea")
+      .length > 0
+  ) {
+    throw new Error("LabeledValue cannot contain an editable element.");
+  }
```

Comment:
We'll want to avoid reading from a `ref` during render (I think we have a lint rule for this), so you can put this inside a `useEffect `.